src/expr/src/expr/build.rs

```diff
@@ -70,6 +71,15 @@ pub fn build_from_prost(prost: &ExprNode) -> Result<BoxedExpression> {
     }
 }
 
+/// Build an expression from protobuf with wrappers.
+pub fn build_from_prost(prost: &ExprNode) -> Result<BoxedExpression> {
+    let expr = build_from_prost_inner(prost)?;
+
+    let checked = CheckedExpression(expr);
+
+    Ok(checked.boxed())
```

Comment:
Why insert empty line between every lines😁

Reply:
Will soon be refactored in the next PR. 🤣